### ISSUE TITLE
 class progression
 ~upvote=0|downvote=0

### ISSUE THREAD

**Issue by LuisEngelniederhammer** [OutlawByteStudios/KingdomsDiscordBot#29]
What about a class system kind of like persistent world, but instead of paying to upgrade to a class, have a branching tier system like native troop trees. Everyone starts as a serf, and then can choose one of these paths to go down:
-> huntsman -> archer -> master archer
-> militia -> soldier -> knight/cavalry
-> healer -> doctor
-> apprentice smith -> master smith
etc

Players can go down their chosen path by doing actions associated with the path they are taking. I think this could be neat for roleplay servers because it would give players a way of working up to their chosen role ingame if they don't want to apply on the forums or whatever.

 >🏰 Discord Suggestion from Journeyman@5690

**Comment by Duckie427** [comment]
I agree with this idea, there were many lower tier classes in PW which were simply ignored due to their inferiority. I believe it would  be a nice change and progression, perhaps it could be reset after you die that way you could add an extra layer of stakes.
